### PR TITLE
Create tests for Milestone service

### PR DESCRIPTION
**Pull request by MadLamprey**
### Summary:

Fixes part of #87

#### Type of change:
- 🧪 Tests Update

### Changes Made:

- Added test cases for `milestone.service.ts`

### Proposed Commit Message:

```
Create tests for Milestone service

Currently, there are no tests written for Milestone service.
Tests are necessary to ensure it is functioning as expected.

Let's create tests for milestone service.
```
###

### CODE REVIEW

**Review by NereusWB922 — DISMISSED** [review]
LGTM!

File: tests/services/milestone.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    });
+  });
+
+  describe('MilestoneService: parseMilestoneData()', () => {
+    it('should parse milestone data correctly', () => {
+      const mockMilestones = [{ title: 'Milestone 2' }, { title: 'Milestone 1' }];
+      const parsedMilestones = milestoneService.parseMilestoneData(mockMilestones);
+
+      for (const milestone of parsedMilestones) {
+        expect(milestone instanceof Milestone).toBeTrue();
```
Comment (luminousleek) [inline tests/services/milestone.service.spec.ts:47]: Can use the jasmine matcher `toBeInstanceOf` here.

**Review by luminousleek — CHANGES_REQUESTED** [review]
Small nit, other than that the code LGTM. Please wrap your commit message at 72 characters too.

**Review by luminousleek — APPROVED** [review]
LGTM! Thanks for making the change.

**Pull request merged**